src/OutlookGoogleCalendarSync/OutlookOgcs/OutlookCalendar.cs

```diff
@@ -540,7 +540,7 @@ public enum Service {
                             Boolean foundAttendee = false;
                             try {
                                 recipient = recipients[r];
-                                if (recipient.Name == ai.Organizer) continue;
+                                if (recipient.Name == GetEventOrganizer(ai)) continue;
```

Comment:
The wrappers in this `AddAttendees` section aren't strictly needed as the GUI won't allow syncing of attendees if access to the GAL is blocked by corporate policy.